apps/web/src/app/vault/individual-vault/add-edit.component.ts

```diff
@@ -20,8 +20,8 @@ import { CollectionService } from "@bitwarden/common/vault/abstractions/collecti
 import { FolderService } from "@bitwarden/common/vault/abstractions/folder/folder.service.abstraction";
 import { PasswordRepromptService } from "@bitwarden/common/vault/abstractions/password-reprompt.service";
 import { CipherType } from "@bitwarden/common/vault/enums/cipher-type";
-import { Fido2KeyView } from "@bitwarden/common/vault/models/view/fido2-key.view";
-import { LoginUriView } from "@bitwarden/common/vault/models/view/login-uri.view";
+
+import { Launchable } from "./../../../../../../libs/common/src/vault/interfaces/launchable";
```

Comment:
issue: woops auto import! :D
```suggestion
import { Launchable } from "@bitwarden/common/vault/interfaces/launchable";
```

Reply:
oops, I missed this. Thanks